website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Data Vault 2.0 with dbt Cloud"
+description: "When to use, and when not to use Data Vault 2.0 data modeling, and why dbt Cloud is a great choice"
+slug: data-vault-with-dbt-cloud
+
+authors: [rastislav_zdechovan, sean_mcintyre]
+
+tags: [data vault] 
+hide_table_of_contents: false
+
+date: 2023-06-30
+is_featured: true
+---
+
+Data Vault 2.0 is a data modelling technique designed to help scale large data warehousing projects. It is a rigid, prescriptive system detailed vigorously in [a book](https://www.amazon.com/Building-Scalable-Data-Warehouse-Vault/dp/0128025107) that has become the bible for this technique.
+
+So why Data Vault? Have you experienced a data warehousing project with 50+ data sources, with 25+ data developers working on the same data platform, or data spanning 5+ years with two or more generations of source systems? If not, it might be hard to initially understand the benefits of Data Vault, and maybe [Kimball modelling](https://docs.getdbt.com/blog/kimball-dimensional-model) is better for you. But if you are in _any_ of the situations listed, then this is the article for you!
+
+Here’s an analogy to help illustrate Data Vault:
+
+Think of a city’s water supply. Each house does not have a pipe directly from the local river: there is a dam and a reservoir to collect water for the city from all of the sources – the lakes, streams, creeks, glaciers – before the water is redirected into each neighbourhood and finally into each home’s taps.
+
+A new development in the city? No problem! Just hook up the new pipes to the reservoir! Not enough water? Just find another water source and fill up the reservoir.
+
+Data Vault is the dam and reservoir: it is the well-engineered data model to structure an organisation’s data from source systems for use by downstream data projects – rather than each team collecting data straight from the source. The Data Vault data model is designed using a few well-applied principles, and in practice, pools source data so it is available for use by all downstream consumers. This promotes a scalable data warehouse through reusability and modularity.
```

Comment:
```suggestion
Data Vault is the dam and reservoir: it is the well-engineered data model to structure an organization’s data from source systems for use by downstream data projects – rather than each team collecting data straight from the source. The Data Vault data model is designed using a few well-applied principles, and in practice, pools source data so it is available for use by all downstream consumers. This promotes a scalable data warehouse through reusability and modularity.
```